addressbook/addressbook.sh

```diff
@@ -0,0 +1,267 @@
+#!/bin/ash
+
+# Functions
+format_echo()
+{
+	echo "$1" | sed 's/:/ | /g'
+}
+
+usage()
+{
+	echo "usage: addressbook [OPTIONS]"
+	echo
+	echo "OPTIONS"
+	echo "	-h"
+	echo "		Display usage"
+	echo "	-a"
+	echo "		Display all entries."
+	echo "	-s NAME PHONE EMAIL"
+	echo "		Save entrie with name, phone naumber and email address."
+	echo "	-f SEARCH"
+	echo "		Find entrie/entries by the given input."
+	echo "	-e INDEX FIELD_INDEX CHANGE"
+	echo "		Replace entrie's field with CHANGE"
+	echo "	-r INDEX"
+	echo "		Removes entrie by the given INDEX"
+}
+
+save_entrie()
+{
+	if  grep -q "${1}:${2}:${3}" data.txt
+	then
+		echo "Entrie already exists..."
+		echo
+	else
+		echo "$1:$2:$3" >> data.txt
```

Comment:
The database only at the first add will be created. This can be produce error messages in other functions.